stories/Components/Forms/Checkbox/Checkbox.jsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import './checkbox.scss';
 
-export const Checkbox = ({ label, value, id }) => (
+export const Checkbox = ({ label, value, arialabel, id }) => (
   <div className="form-check">
-    <input type="checkbox" id={id} name={id} value={value}/>
+    <input type="checkbox" id={id} name={id} value={value} {...(arialabel === '' ? {} : {'aria-label': arialabel})}/>
```

Comment:
condition should be like this @Shubham-Sharma-77  
`<input type="checkbox" id={id} name={id} value={value} {...(arialabel != '' ?  {'aria-label': arialabel})}/>`